src/styles/theme.scss

```diff
@@ -83,6 +85,10 @@ html[data-theme='dark'] {
 	--panel-title: #2d3036;
 
 	// rest
+	--info-color: #71a2ff;
```

Comment:
i dont think that is true, there are only a few places where this hex color is used
it might be an error